node_modules/oae-tenants/config/config.js

```diff
@@ -61,6 +61,16 @@ var timezoneConfigValues = _.chain(TZ.getZones())
 
 module.exports = {
     'title': 'OAE Tenant Module',
+    'instance': {
+        'name': 'Instance Information',
+        'description': 'Information about the current instance',
+        'elements': {
+            'instanceName': new Fields.Text('Instance name', 'The name of the current instance', '', {'tenantOverride': false}),
+            'instanceURL': new Fields.Text('Instance URL', 'The URL of the main instance website', '', {'tenantOverride': false}),
+            'hostingOrganisation': new Fields.Text('Hosting organisation', 'The name of the organisation hosting the current instance', '', {'tenantOverride': false}),
+            'hostingOrganisationURL': new Fields.Text('Hosting organisation URL', 'The URL of the hosting organisation website', '', {'tenantOverride': false})
+        }
+    },
```

Comment:
A quick check shows we don't have much of a precedence at least in config keys. I feel like using american spelling for dev-facing public API things makes usage more predictable, but it could just be because I've developed more in NA. What do you think?